packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -63,7 +63,7 @@ $: onboardingItems = onboardings;
 let activeStep: ActiveOnboardingStep;
 let activeStepContent: OnboardingStepItem[][];
 
-$: executing = false;
+let executing = false;
```

Comment:
it's changing the behavior as it won't reset executing to false if some attributes have changed